### PR TITLE
Adds more arguments to trigger a double tap

### PR DESCRIPTION
**Pull request by phil-pona**

### CODE REVIEW

File: gpio_handler/gpio_handler.py [hunk 1]
```diff
@@ -127,12 +142,21 @@ def draw_beer(wait=t_large_beer):
...
         print("Choice: Large beer")
-        draw_beer(t_large_beer)
+        draw_beer(r_ch1, t_large_beer)
     elif args.products == "SMALL":
         print("Choice: Small beer")
-        draw_beer(t_small_beer)
+        draw_beer(r_ch1, t_small_beer)
+    elif args.products == "LEFT_TAP":
+        print("Choice: left tap")
+        draw_beer(r_ch2,t_left_tap)
```
Comment (guggero) [inline gpio_handler/gpio_handler.py]: Shouldn't this be `draw_beer(r_ch1, t_left_tap)`?

File: gpio_handler/gpio_handler.py [hunk 2]
```diff
@@ -127,12 +142,21 @@ def draw_beer(wait=t_large_beer):
...
+    elif args.products == "RIGHT_TAP":
+        print("Choice: right tap")
+        draw_beer(r_ch2,t_right_tap)
```
Comment (guggero) [inline gpio_handler/gpio_handler.py:160]: nit: space after comma

**Review by guggero — CHANGES_REQUESTED** [review]
Very nice, thanks!
Just two small things that I found.

**Review by id101010 — COMMENTED** [review]
Looks good :)

**Pull request merged**